rfc6962-bis.xml

```diff
@@ -1201,7 +1201,7 @@ entry specified by <spanx style="verb">start</spanx>.
       </section>
       <section title="TLS Extension" anchor="tls_transinfo_extension">
         <t>
-          Provided that a TLS client includes the <spanx style="verb">transparency_info</spanx> extension type in the ClientHello, the TLS server MAY include the <spanx style="verb">transparency_info</spanx> extension in the ServerHello with <spanx style="verb">extension_data</spanx> set to a <spanx style="verb">TransItemList</spanx>. The TLS server MUST NOT process or include this extension when a TLS session is resumed, since session resumption uses the original session information.
+          Provided that a TLS client includes the <spanx style="verb">transparency_info</spanx> extension type in the ClientHello, the TLS server MAY include the <spanx style="verb">transparency_info</spanx> extension in the ServerHello with <spanx style="verb">extension_data</spanx> set to a <spanx style="verb">TransItemList</spanx>. The TLS server MUST NOT process or include this extension when a TLS session is resumed, since session resumption uses the original session information. The TLS server SHOULD also ignore any data sent by the client in the <spanx style="verb">transparency_info</spanx> extension.
```

Comment:
I think it would flow better to make the point about ignoring the client's extension_data before making the point about session resumption.